scripts/publish.sh

```diff
@@ -201,10 +205,14 @@ publish_to_public_ecr() {
 		create_manifest_list public.ecr.aws/aws-observability/aws-for-fluent-bit ${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
 		aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
 		create_manifest_list public.ecr.aws/aws-observability/aws-for-fluent-bit "latest" ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_debug public.ecr.aws/aws-observability/aws-for-fluent-bit "debug-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_debug public.ecr.aws/aws-observability/aws-for-fluent-bit "debug"-${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
 
 		create_manifest_list_init public.ecr.aws/aws-observability/aws-for-fluent-bit "$init"-${AWS_FOR_FLUENT_BIT_VERSION} ${AWS_FOR_FLUENT_BIT_VERSION}
 		aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
 		create_manifest_list_init public.ecr.aws/aws-observability/aws-for-fluent-bit "init-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
+		create_manifest_list_init_debug public.ecr.aws/aws-observability/aws-for-fluent-bit "init-debug-latest" ${AWS_FOR_FLUENT_BIT_VERSION}
```

Comment:
Should we use $init here? I'm fine if you don't. This would just be for consistancy

Reply:
above this line, we used "init-latest" before, so I followed it with "init-debug-latest"